### PR TITLE
README.rst refinement

### PR DESCRIPTION
**Pull request by maxoja**
- Inserted usage example replicated from official tutorial to illustrate the usefulness of this tool for new comers
- Uniformed the pattern of bullet list as well as capitalization along all sections to improve readability
- Fixed some small grammar errors for better precision in meanings

Please review for correctness
Thank you!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,47 +1,61 @@
 .. image:: https://secure.travis-ci.org/cobrateam/splinter.svg?branch=master
    :target: http://travis-ci.org/cobrateam/splinter
 
-
 ++++++++++++++++++++++++++++++++++++++++++++++++
-splinter - python tool for test web applications
+Splinter - python tool for testing web applications
```
Comment (andrewsmedina) [inline README.rst]: we use the splinter name as lowercase.

**Comment by maxoja** [comment]
glad for your guidance, edited as requested ^^

**Comment by andrewsmedina** [comment]
@maxoja can you sync your branch with the master?

**Comment by maxoja** [comment]
@ableade Ok, I merged patch-1 to master on my forked.

**Comment by andrewsmedina** [comment]
@ableade  You should resolve the conflicts to make easier to me to integrate this pr.

**Comment by maxoja** [comment]
Sorry that I misunderstood your guide. Now resolved the conflict. Thanks

**Comment by andrewsmedina** [comment]
@maxoja  thank you!

**Pull request merged**